exporter/zipkin/lib/opentelemetry/exporter/zipkin/exporter.rb

```diff
@@ -101,8 +101,8 @@ def encode_spans(span_data)
           span_data.map! { |span| Transformer.to_zipkin_span(span, span.resource) }
         end
 
-        def around_request
-          OpenTelemetry::Common::Utilities.untraced { yield }
+        def around_request(&block)
+          OpenTelemetry::Common::Utilities.untraced(&block)
```

Comment:
There was a time when the original code here (`def foo; bar { yield }; end`) was more efficient with MRI than the explicit block reference. IDK if that's still true, but it is the rationale for using the old form.

Reply:
I will look, thanks for the context.

Reply:
I didn't see anything conclusive (didn't look too hard, tbh), but changed it back to how it was: 9ccbeefdf95ad414cd789712a4fadf4ae1aff8f8